website/blog/2021-05-09-product-update-10-7-whats-new.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "WaveMaker Releases 10.7 Version"
+author: Swetha Kundaram
+---
+
+
+WaveMaker releases the **10.7.0 update** on 09 May 2021. For the detailed list of features and fixes, see [WaveMaker release notes 10.7.0](/learn/wavemaker-release-notes/v10-7-0).
+
+Take a peek into what's new in this release. 
+
+<!-- truncate -->
+
+## What's New
+
+- **[Working in branches](/learn/wavemaker-release-notes/v10-7-0#work-in-branches)** offers more than just tracking developers' changes. Organize branches to plan features and hotfixes simultaneously, independently. Applies to WMO Teams subscribed customers.  
+
+
+- While support for branching requires enhancements to **[how VCS operates](/learn/wavemaker-release-notes/v10-7-0#vcs-enhancements)**, including how you manage projects, groups, and organizations. See how these changes your [current workflow](/learn/wavemaker-release-notes/v10-7-0#vcs-enhancements) if you are a member of WMO Teams.
+
+
+- Are you already a part of the Amazon ecosystem? In that case, we have good news for you. With the **[support of the Redshift database](/learn/wavemaker-release-notes/v10-7-0#support-for-red-shift-database)** in WaveMaker, get speedy, high performance, horizontally scalable, with the ability to add massive capacity storage—all in the cloud using clustering model.
+
+
+- Are you using **[SAP HANA](/learn/wavemaker-release-notes/v10-7-0#support-for-sap-hana-database)** in your SAP modules? With SAP HANA implementation in WaveMaker, import the SAP HANA database into your WaveMaker application for seamless integration, supporting the columnar data model. It improves flexibility, performance and uses an in-memory store with the read and editable modes.
+
+- Limitations to access public servers apply to most organizations. We understand. **[UI Artifacts](/learn/wavemaker-release-notes/v10-7-0#ui-artifacts-now-published-to-npm)** publish to the npm package manager. Incorporate UI Artifacts and simplify CI-CD pipeline using standard npm repo without worrying about firewall restrictions safely. 
```

Comment:
@vibhus this needs rewritten

Reply:
Addressed the UI Artifacts section with the edit provided by @vibhus